torch/_dynamo/variables/builtin.py

```diff
@@ -660,6 +660,10 @@ def never(tx: "InstructionTranslator", a, b):
                             (VariableTracker, SymNodeVariable),
                             op_var._comparison_with_symnode,
                         ),
+                        (
+                            (variables.ExceptionVariable, variables.ExceptionVariable),
+                            lambda tx, l, r: ConstantVariable(op(l, r)),
+                        ),
```

Comment:
I think this is safe to do on ExceptionVariable because we do not use SourcelessBuilder on it. Maybe we should just add an assertion into SourcelessBuilder that we don't see one?